src/Catrobat/Controller/Api/ExampleController.php

```diff
@@ -0,0 +1,120 @@
+<?php
+
+namespace App\Catrobat\Controller\Api;
+
+use App\Entity\ExampleProgram;
+use Doctrine\ORM\NonUniqueResultException;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\Routing\Annotation\Route;
+use App\Repository\ExampleRepository;
+use App\Catrobat\Services\ExampleImageRepository;
+
+
+/**
+ * Class ExampleController
+ * @package App\Catrobat\Controller\Api
+ */
+class ExampleController extends AbstractController
+{
+
+  /**
+   * @Route("/api/projects/example.json", name="api_example_programs",
+   *   defaults={"_format": "json"}, methods={"GET"})
+   *
+   * @param Request $request
+   * @param ExampleImageRepository $image_repository
+   * @param ExampleRepository $repository
+   *
+   * @return JsonResponse
+   * @throws NonUniqueResultException
+   */
+  public function getExampleProgramsAction(Request $request, ExampleImageRepository $image_repository,
+                                            ExampleRepository $repository)
+  {
+    return $this->getExamplePrograms($request, false, $image_repository, $repository);
+  }
+
+
+  /**
```

Comment:
it would be better to just use one route and retrieve ios from the request.
like it will be done for featured projects

![image](https://user-images.githubusercontent.com/40868718/72618938-81fec580-393c-11ea-8008-e8bce72f93bf.png)

@HCrane you might have to add this API call to your Swagger API.
Or should this be changed later with the API rework? @HCrane